executable/te.ksy

```diff
@@ -0,0 +1,125 @@
+meta:
+  id: te
```

Comment:
We have no namespaces or anything for these, so it's preferable not use very short acronyms, especially if they can be ambiguous for top-level IDs. Please consider something like `uefi_te`?